metricsaggregator/src/main/resources/logback.xml

```diff
@@ -23,7 +23,7 @@
         </encoder>
     </appender>
 
-    <root level="error">
+    <root level="info">
```

Comment:
intentional?

Reply:
Yeah, I wanted to see the INFO logs otherwise its eerily quiet on the CLI :grin: